### PR TITLE
add sleep() for fix compile error on windows

### PR DESCRIPTION
**Pull request by skyformat99**
add sleep() for fix compile error on windows.

### CODE REVIEW

File: include/edge_utils_win32.h [hunk 1]
```diff
@@ -21,9 +21,12 @@
 
 #ifdef WIN32
 
+#define WIN32_LEAN_AND_MEAN
```
Comment (lucaderi) [inline include/edge_utils_win32.h:24]: What is this define? Why is necessary?

Comment (skyformat99) [reply include/edge_utils_win32.h:24]: ref1:   https://issues.apache.org/jira/browse/THRIFT-2150
When both Windows.h and Winsock2.h are included, it is easy to cause compilation conflicts.
ref2:  https://msdn.microsoft.com/en-us/library/windows/desktop/aa383745(v=vs.85).aspx
   the explanation at the end of the article.

**Comment by lucaderi** [comment]
Thanks

**Comment by fengdaolong** [comment]
If it must be "#define WIN32_LEAN_AND_MEAN", why not do it like below.

#define WIN32_LEAN_AND_MEAN
#include <windows.h>
#undef WIN32_LEAN_AND_MEAN

**Pull request merged**